src/engines/ioda/src/ioda/Engines/HH/HH/Handles.h

```diff
@@ -137,6 +137,7 @@ struct InvalidHDF5Handle {
 namespace Closers {
 struct CloseHDF5Attribute {
   static inline void Close(hid_t h) {
+    if(h == H5I_INVALID_HID) return;
```

Comment:
@rhoneyager could you quickly advise us on whether it is a good idea to put in these safety checks (to prevent the closers from acting on an invalid hdf5 object)? Would you agree that if the hid is H5_INVALID_HID, then there is no object to be closed and it's okay to forego calling the associated hdf5 close function. Thanks!

Reply:
FYI, the JEDI executables completed successfully. We got the output ioda files, the updated analysis files and we can cycling for one week.

Curious about SkyLab: does it have those unnecessary "error" messages after a successful JEDI run?